ActiveRecord notable changes added to 7.1 release note [skip ci] (#49357)
rails/rails: guides/source/7_1_release_notes.md

*   Add `TestFixtures#fixture_paths` to support multiple fixture paths.

*   Add `authenticate_by` when using `has_secure_password`.

> *   Add `update_attribute!` to `ActiveRecord::Persistence`, and it is similar to `update_attribute`,

Reviewer: ```suggestion
*   Add `update_attribute!` to `ActiveRecord::Persistence`, which is similar to `update_attribute`
```
